index.html

```diff
@@ -8,12 +8,83 @@
     <link rel="stylesheet" href="css/normalize.css">
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
 </head>
 <body>
+    <header>
+        <img src="img/karma-logo.svg" alt="Karma logo" >
+        <nav class="topnav" id="myTopnav">
+            <a href="#">Meet Karma</a> 
+            <a href="#">How it works</a>
+            <a href="#">Store</a>
+            <a href="#">Blog</a>
+            <a href="#">Help</a>
+            <a href="#">Login</a>
```

Comment:
```suggestion
        <nav class="topnav" id="myTopnav">
            <ul>
                <li><a href="#">Meet Karma</a></li>
                <li><a href="#">How it works</a></li>
                <li><a href="#">Store</a></li>
                <li><a href="#">Blog</a></li>
                <li><a href="#">Help</a></li>
                <li><a href="#">Login</a></li>
            </ul>
```
In a `nav` we usually have a list of links, and because it's a list we typically use `ul` or `ol` for the links. We can see this in the documentation for `nav`:

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav